### ISSUE TITLE
Failure to resolve

### ISSUE THREAD

**Issue by Reedyuk** [segmentio/analytics-kotlin#66]
**Describe the bug**
Unable to resolve the library using gradle.

**To Reproduce**
Add dependancy build gradle kts
```
val androidMain by getting {
            dependencies {
                implementation("com.github.segmentio.analytics-kotlin:android:1.2.0")
            }
        }
```
I even tried with 
```
maven("https://repository-master.mulesoft.org/nexus/content/groups/public/")
```
Shows: Failed to resolve: com.github.segmentio.analytics-kotlin:android:1.2.0

**Expected behavior**
Able to fetch lib

Do you have a working example where it is fetching from maven central or mulesoft?

**Comment by Reedyuk** [comment]
The documentation is wrong, the correct package name is:
```
implementation("com.segmentio.analytics.kotlin:android:1.2.0")
```
The documentation should be updated to reflect this.

**Comment by Reedyuk** [comment]
Actually scrap that, it still doesn't work. 
Its actually quite strange because when publishing locally the folder structure is as above.

**Comment by wenxi-zeng** [comment]
@Reedyuk please try the following:
```
com.github.segmentio:analytics-kotlin:1.2.0
```

thanks for reporting the issue. we'll get the documentation fixed. we're migrating to maven central, you'll soon be able to use it in the way you suggested.

**Comment by tagrudev** [comment]
@wenxi-zeng this still doesn't work :/

**Comment by wenxi-zeng** [comment]
@tagrudev you should be able to use it in the following way for version 1.3.0 and older
```
com.github.segmentio:analytics-kotlin:1.2.0
```

can you share your usage, so we have a better idea?

**Comment by tagrudev** [comment]
Sorry @wenxi-zeng I should have added more information, so in my `build.gradle.kts` I have the following

```
implementation("com.github.segmentio:analytics-kotlin:1.3.0")
```

I have added the jitpack to my repositories as well

```
maven(url = "https://jitpack.io")
```

This still results in 
<img width="568" alt="Screenshot 2022-01-11 at 9 17 06" src="https://user-images.githubusercontent.com/438405/148897832-c2195265-b449-41f2-918e-6de7eb0f4516.png">

**Comment by wenxi-zeng** [comment]
hey @tagrudev sorry for the late response. I have no idea why your setup doesn't work. it seems right to me. but we have just released our new version 1.4.2 to maven and it is ready to be used. our [public doc](https://segment.com/docs/connections/sources/catalog/libraries/mobile/kotlin-android/) has been updated as well. please check out the doc to see if the new maven setup works for you. thanks!

**Comment by tagrudev** [comment]
Using 1.4.2 and only mavenCentral works now. Thank you.
